src/test/java/com/xiaomi/infra/pegasus/rpc/async/MetaSessionTest.java

```diff
@@ -327,7 +327,7 @@ public void testDNSMetaUnavailable() {
                 .build());
     MetaSession metaMock = Mockito.spy(manager.getMetaSession());
     List<ReplicaSession> metaList = metaMock.getMetaList();
-    metaList.remove(0); // del the "localhost:34601"
+    metaList.clear(); // del the "localhost:34601"
```

Comment:
irrelevant change

Reply:
must update for github-action. The test is for `DNS resolve`, travis resolve localhost:34601 to one ipport, but github-action container resolve mutli-ipport, so must use clear but remove(0)